server/php-slim/index.php

```diff
@@ -68,7 +68,7 @@
   $checkout_session = \Stripe\Checkout\Session::create([
     'success_url' => $domain_url . '/success.html?session_id={CHECKOUT_SESSION_ID}',
     'cancel_url' => $domain_url . '/canceled.html',
-    'payment_method_types' => ['card'],
+    'payment_method_types' => $_ENV['PAYMENT_METHOD_TYPES'],
```

Comment:
I have a feeling this might "just work", but also we might want to split this one too? 


explode(",", $_ENV['PAYMENT_METHOD_TYPES'])